data/pilots/rebel-alliance/t-65-x-wing.json

```diff
@@ -307,6 +307,262 @@
       "extended": true,
       "keywords": ["X-wing"],
       "epic": true
+    },
+    {
+      "name": "Biggs Darklighter",
+      "caption": "Battle of Yavin",
+      "initiative": 3,
+      "limited": 1,
+      "cost": 5,
+      "standardLoadout": true,
+      "xws": "biggsdarklighter-battleofyavin",
+      "ability": "During the System Phase, you may choose 1 friendly ship at range 1. If you do, treat your initiative as equal to the chosen ship's initiative until the end of the Activation Phase.",
+      "shipAbility": {
+        "name": "Hope",
+        "text": "After another friendly ship at range 0-3 is destroyed, you may perform a [Focus] or [Boost] action."
+      },
+      "upgrades": [
+        {
+          "title": "Attack Speed",
+          "slots": ["Talent"],
+          "ability": "After you fully execute a [3 [Straight]] or [4 [Straight]] maneuver, you may perform a boost using the [1 [Straight]] template. (This is not an action)."
+        },
+        {
+          "title": "Selfless",
+          "slots": ["Talent"],
+          "ability": "While another friendly ship at range 0-1 defends, before the Neutralize Results step, if you are in the attack arc, you may suffer 1 [Critical Hit] damage to cancel 1 [Critical Hit] result.",
+          "limited": 1
```

Comment:
Selfless has a limited dot on Bigg's card. That seems like an accident? Haven't seen any discussion on it anywhere, but also have had my head under a rock. Anyone know if this is supposed to be limited?